settings/src/main/java/com/islamversity/settings/SettingsView.kt

```diff
@@ -0,0 +1,29 @@
+package com.islamversity.settings
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.preference.PreferenceFragmentCompat
+import com.example.search.R
+
+class SettingsView : AppCompatActivity() {
```

Comment:
Like rest of views, `SettingsView `must be a extended from [CoroutineView](https://github.com/islamversity/Reyan/blob/ed2e2546748038e15fa3dc014526d7759fe2d1d4/base/src/main/java/com/islamversity/base/CoroutineView.kt)